src/simulator/shape.py

```diff
@@ -13,3 +14,13 @@ def bounding_box(self, position: Position):
 
     def dump_info(self):
         pass
+
+    def rotation(x: float, y: float, z: float, phi: float, theta: float):
```

Comment:
Please add type hint for the return type, i.e. `list[float]`

Reply:
+ Could you add some comments for this function?

Reply:
sorry, I forgot comment. I will add comments and type hint.